Xamarin.Forms.Platform.UAP/TabbedPageRenderer.cs

```diff
@@ -325,11 +339,17 @@ void UpdateBarBackgroundColor()
 			if (barBackgroundColor == _barBackgroundColor) return;
 			_barBackgroundColor = barBackgroundColor;
 
+			ApplyBarBackgroundColor();
+		}
+
+		void ApplyBarBackgroundColor(bool force = false)
+		{
 			var controlToolbarBackground = Control.ToolbarBackground;
-			if (controlToolbarBackground == null && barBackgroundColor.IsDefault) return;
+			if (controlToolbarBackground == null && _barBackgroundColor.IsDefault) return;
```

Comment:
Now I can't stand the single line `if return` :X

Reply:
I hate those, too. Not sure how they got past the Great Reformatting of '16.